### DISCUSSION TITLE
user exits but I can't log in

### DISCUSSION THREAD

**Discussion by qendrimkryeziu** [vercel/community#5073] [Account Issue]
### Question

Hi,

I have registered for the first time in vercel and I have a problem in logging in.

When I'm trying to log in, the error that shows up is: "There is already an account associated with your GitHub email address. Please login with your email and password then link your GitHub account. Login with Email"

But when I'm trying to log in with my email address what shows up is:" There is no Vercel account associated with this email address. Sign up?"

Thank you!

**Comment by amyegan** [comment]
Do you get the same error if you go to [vercel.com/login](https://vercel.com/login) directly and continue with GitHub?

**Reply by qendrimkryeziu** [reply]
This is the error when I try directly with github to login: There is already an account associated with your GitHub email address. Please login with your email and password then link your GitHub account. Login with Email
